crates/proto/go/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "proto_go"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
i think most of these deps can be removed

Reply:
i'm getting:
```
All deps seem to have been used.
```

Reply:
we export `Version` from the proto core crate, so you can use that instead of lenient_semver directly

Reply:
Cool, will update in the next PR. I have have the other go stuff cooking on another branch